cmd/podman/create_cli.go

```diff
@@ -74,6 +74,49 @@ func addWarning(warnings []string, msg string) []string {
 	return append(warnings, msg)
 }
 
+func parseMounts(mounts []string) error {
```

Comment:
Can't you combine this function and the function below to create the mounts struct into one function?
 func (c *CreateConfig) GetFSMounts(specMounts []spec.Mount) ([]spec.Mount, error) {

Reply:
There are little more required to just merge these functions, as [`parseMount()`](https://github.com/containers/libpod/pull/1432/files#diff-958bbd628fbbafabf2cdfc21a7dbe118R80) uses other functions present in `create_cli.go` file, whereas [`GetFSMounts()`](https://github.com/containers/libpod/pull/1432/files#diff-cebb6d9e2dd46ffe9665b195619c5c13R226) has some common logic that is shared with functions of [`/pkg/createconfig.go`](https://github.com/containers/libpod/blob/7b4489a945bd8eddfd791bd5e2e888ee348c9ea5/pkg/spec/createconfig.go#L150) like `GetVolumeMounts()`.

I think, reorganizing code for various mount options (`-v` , `--mount`, `--volume-from`) to some common function will be better, not can be done in separate PR.